Assignment2/Extract_Bits/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+
+/// <summary>
+/// Gets the number, start and stop bits then extracts the bits and provides the decimal value of it.
+/// </summary>
+class BitsExtraction{
```

Comment:
Access scope for class missing in all .cs files

Reply:
Added access scope for all classes.